src/liff/lib.js

```diff
@@ -75,3 +84,18 @@ export const gql = (query, ...substitutions) => variables => {
       return resp;
     });
 };
+
+export const ClientOnlyCheck = () => {
```

Comment:
I think we can go for verb + lower cased charactors, like `assertInClient`.

Additional descriptions can also be added:
```javascript
/**
 * Prevent users proceed with external browsers.
 * Useful when the following process involves functions only available within LINE client,
 * such as invoking `liff.sendMessage()`.
 */
```